src/main/java/com/mraof/minestuck/block/machine/AlchemiterBlock.java

```diff
@@ -79,6 +80,108 @@ public void onRemove(BlockState state, Level level, BlockPos pos, BlockState new
 		}
 	}
 	
+	/**
+	 * Destroys and then checks which blocks are connected to the given block in the multiblock structure, then repeats the process for those blocks, until the entire structure is destroyed.
+	 * @param state The blockstate of the block being currently destroyed.
+	 * @param level The server level/world
+	 * @param pos The position of the block currently being destroyed.
+	 */
+	@Override
+	public void findAndDestroyConnected(BlockState state, Level level, BlockPos pos)
```

Comment:
There are two things I'd like changed with this and other such functions:
- When looking at a connected block, the rotation of the connected block should be checked to ensure that it actually belongs to the same alchemiter, rather than just *some* alchemiter.
- This hardcodes the block to use `MSBlocks.ALCHEMITER`, and not respecting any changes to block placements made in the multiblock. A more general solution utilizing block placements from the multiblock object would be good, which would then appropriately be written in the multiblock class.

However implementing these changes may be a bit more complicated than what has been made so far. If you don't think you could implement it yourself, feel free to poke me and I can look into implementing it myself.